protocol/x/listing/types/constants.go

```diff
@@ -1,6 +1,8 @@
 package types
 
 const (
+	MinPriceChangePpm_LongTail uint32 = 800
```

Comment:
nit: add comment that these constants correspond to long-tail markets

Reply:
The constants specific to long-tail markets have that noted in the suffix to the name